src/restconfig/src/main/java/org/geoserver/rest/catalog/StyleController.java

```diff
@@ -225,7 +225,7 @@ private UriComponents getUriComponents(String name, String workspace,
         return uriComponents;
     }
 
-    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"})
+    @GetMapping(path = {"/styles/{styleName}", "/workspaces/{workspaceName}/styles/{styleName}"}, produces = {MediaType.ALL_VALUE})
```

Comment:
We wrote a test case for ysld, but I am starting to fear we did not test for format conversion.